.mage/js_sdk.go

```diff
@@ -191,12 +196,18 @@ func (k JsSDK) DefinitionsClean(context.Context) error {
 
 // Link links the local sdk package via `yarn link` to prevent caching issues.
 func (k JsSDK) Link() error {
+	if nodeEnv := os.Getenv("NODE_ENV"); nodeEnv != "development" {
```

Comment:
This is incorrect - SDK link is always required, otherwise frontend dependency cannot be found for e.g. `js:lint` and it should work regardless of `NODE_ENV`, cc @bafonins

Reply:
To be more precise, we indeed do not need to link the js sdk in production mode, but we must make sure that the sdk is built before the rest of the frontend. Otherwise, webpack build fails as it cannot find the `ttn-lw` dependency. I took similar approach in https://github.com/TheThingsNetwork/lorawan-stack/pull/2751